config/feature.json

```diff
@@ -0,0 +1,5 @@
+{
+    "features": {
+        "FEATURE_1_ACTIVE": false
```

Comment:
Can we remove ACTIVE, I don't think it is necessary. It should be enough to have a feature_name label and whether it is enabled or disabled